packages/cosmic-swingset/lib/ag-solo/vats/repl.js

```diff
@@ -13,6 +13,9 @@ export function stringify(
   depth = 0,
 ) {
   if (Object(value) !== value) {
+    if (typeof value === 'bigint') {
+      return `${value}n`;
+    }
     return JSON.stringify(value, null, spaces);
   }
```

Comment:
As the test that comes first, this code will fail on symbols. That's not the concern of this PR but is worrisome.

Since it fails in JSON.stringify, it won't fail with an error, but rather by ignoring the symbol. This code will also not work on `undefined`, `NaN`, `Infinity`, or `-Infinity`, all of which are primitive values that don't encode in JSON.

Could you file an issue? Thanks.

Reply:
bug report is #2278